website/docs/r/service_instance.html.markdown

```diff
@@ -35,7 +35,9 @@ The following arguments are supported:
 * `space` - (Required, String) The ID of the [space](/docs/providers/cloudfoundry/r/space.html) 
 * `json_params` - (Optional, String) Json string of arbitrary parameters. Some services support providing additional configuration parameters within the provision request
 * `tags` - (Optional, List) List of instance tags. Some services provide a list of tags that Cloud Foundry delivers in [VCAP_SERVICES Env variables](https://docs.cloudfoundry.org/devguide/deploy-apps/environment-variable.html#VCAP-SERVICES)
-
+* `timeout` - (Optional, Integer) The timeout in seconds, which shall be waited until service instance is created. Default: 60 seconds
+* `async` - (Optional, Bool) 	Set to `true` if the client allows asynchronous provisioning. Please refer to [this](https://github.com/openservicebrokerapi/servicebroker/blob/v2.12/spec.md#synchronous-and-asynchronous-operations) for more information. Default: false 
```

Comment:
Is this flag really necessary ? 
The provider should systematically set the `accepts_incomplete` flag in the [CC API call](http://apidocs.cloudfoundry.org/280/service_instances/creating_a_service_instance.html) and inter from the response status code (202 vs 200) and/or `last_operation.state` response field whether the service instance completion is sync or async

Reply:
Oh I see, the best way is to catch up if the given response is 200 or 202. I'll change the PR and according our the UX recommendations as discussed in [#51](https://github.com/mevansam/terraform-provider-cf/issues/51) beginning of next week.

 Thanks for the review!